cmd/ipfs-cluster-ctl/main.go

```diff
@@ -252,6 +252,21 @@ cluster peers.
 						return nil
 					},
 				},
+				{
+					Name:  "gc",
```

Comment:
My view is to create a new command `ipfs-cluster-ctl repo gc`, but this comes much later (checklist https://github.com/ipfs/ipfs-cluster/issues/628#issuecomment-479527317)

Reply:
@kishansagathiya I disagree that we should have another command just for gc. And I think makes more sense to be under the peers command as this will be broadcast to all peers.

Reply:
I lean towards `ipfs-cluster-ctl peers gc` too.

Reply:
I think of `peers` subcommand as peerset management (ls, rm). This doesn't get us info or change anything in the peerset.

> I think makes more sense to be under the peers command as this will be broadcast to all peers

Out of existing commands `peers` is surely the most appropriate. Many of our commands deals with all peers. With that logic many commands can fit into `peers`. But if I keep this narrow view of `peers`, it doesn't fit as much as `ls` or `rm`. It's not intuitive that `peers` would have `gc`, especially for those users, who have used `ipfs`. But when they see `repo gc`, they would know what it would do.

First I thought of `ipfs-cluster-ctl gc`, but I thought with `repo gc`, it would leave room for other `repo` commands to be added.

Reply:
Also, this feature is different than cluster's primary use(pinset management). With this we would be adding one more resource to manage. Somehow I am inclined to call that resource `repo` than `peers`.

Reply:
At this point it doesn't make sense to add a repo command for one subcommand and it shouldn't be a top level command. It does fit with the model of managing peers and is associated ipfs node

Reply:
Closing this thread for now...

Reply:
So @kishansagathiya does have a point. After all, it's not the cluster peers that are being gc'ed but the ipfs ones (reason why I don't like `repo gc` either). We could do `ipfs-cluster-ctl ipfs gc`, where the `ipfs` subcommand runs commands on the ipfs daemons. Additional commands for the future `ipfs-cluster-ctl ipfs connect` (swarm connect) or... (haven't thought of more).